wallet2/db/query.go

```diff
@@ -0,0 +1,17 @@
+package db
+
+import "fmt"
+
+type QueryOption string
+
+func WithTransactionStatus(status Status) QueryOption {
+	return QueryOption(fmt.Sprintf("WHERE status = %d", status))
```

Comment:
don't use fmt.Sprintf

Reply:
what other option you suggest?

Reply:
please check: https://go.dev/doc/database/sql-injection

Reply:
```go
func Query(query string, args ...any)
```

Reply:
this status comes from `HardCodec Code`, so doesn't make any security issue.

Reply:
rows, err := db.Query("SELECT * FROM user WHERE id = ?", id)